Makefile

```diff
@@ -11,6 +11,11 @@ requirements:
 	pipenv requirements > requirements.txt
 	pipenv requirements --dev > requirements-dev.txt
 
+precommit:
+	pre-commit install
+	pre-commit autoupdate
```

Comment:
Don't we want to add the `autoupdate` command in the `requirements` file and check for changes?
The same way we do with `pipenv` and I have suggested in my comment here: https://github.com/vmware/repository-service-tuf-cli/pull/110#issuecomment-1291935416.

Reply:
Good catch @MVrachev, thanks! I added it in the precommit step of the Makefile but forgot to add it in the requirements as you suggested.
By the way, we will also need to update a bit the description of the update-python-deps.yml workflow, since ```make requirements``` will also include the .pre-commit-config.yaml changes in the PR that it will create. @rdimitrov do you see any problem there, can I open an issue for that?

Reply:
Good catch @MVrachev, thanks! I added it in the precommit step of the Makefile but forgot to add it in the requirements as you suggested because I was hesitating over the workflow side-effects it could have.
We will also need to update a bit the update-python-deps.yml workflow, since ```make requirements``` will also include the .pre-commit-config.yaml changes in the PR that it will create.
We will need to add pre-commit as an install prerequisite and update the description. @rdimitrov do you see any problem there, can I open an issue for that or resolve it in this PR?